remoteit/desktop
# DESK-1012 search cloud new connections

### Description


Cloud search multiple shared device lists for new connection ui

### Test plan



1. Sign in
2. Click new connection
3. Search for a term in any device list
4. You should see results and be able to connect

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1012>
